server/src-rsr/migrations/33_to_34.sql

```diff
@@ -0,0 +1,15 @@
+DO $$ DECLARE
+  r RECORD;
+  BEGIN
+    FOR r IN (SELECT viewname FROM pg_views WHERE schemaname = 'hdb_views' ORDER BY viewname) LOOP
+      EXECUTE 'DROP VIEW IF EXISTS hdb_views.' || quote_ident(r.viewname) || ' CASCADE';
+    END LOOP;
+  END $$;
+
+DO $$ DECLARE
+  r RECORD;
+  BEGIN
+    FOR r IN (SELECT routine_name FROM information_schema.routines WHERE specific_schema = 'hdb_views' ORDER BY routine_name) LOOP
+      EXECUTE 'DROP FUNCTION hdb_views.' || quote_ident(r.routine_name) || '() CASCADE';
```

Comment:
@paf31  Is this required? I am looking at the tests and the `test_server_upgrade` failed on Event Triggers. 

Note that ET functions are present in `hdb_views`. I tried to repro this and indeed it is the case that Event Triggers stop functioning after this upgrade. If I remove this particular routine, they work normally after upgrade. 

I am not certain why this is so. I would have expected that even after removing all the routines during catalog upgrade, the schema building process should have recreated them. We can either investigate this or just not touch the functions of `hdb_views` to get the tests to pass.

Reply:
I've removed it, thanks.